v01-object-or-null/src/main/java/dev/futa/tutorial/pesel/v01/PeselService.java

```diff
@@ -0,0 +1,114 @@
+package dev.futa.tutorial.pesel.v01;
+
+import dev.futa.tutorial.pesel.Gender;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+import static dev.futa.tutorial.pesel.Gender.FEMALE;
+import static dev.futa.tutorial.pesel.Gender.MALE;
+import static java.lang.Character.getNumericValue;
+import static java.lang.Integer.parseInt;
+
+public class PeselService {
+
+  private static final byte PESEL_LENGTH = 11;
+  private static final int[] WEIGHTS_PESEL = {1, 3, 7, 9, 1, 3, 7, 9, 1, 3, 1};
+  private static final Pattern PESEL_PATTERN = Pattern.compile("[0-9]{11}");
+
+  private static final Map<Integer, Integer> YEAR_CODES = new HashMap<>();
+
+  static {
+    YEAR_CODES.put(0, 1900);
```

Comment:
Dla pięciu wartości `ImmutableMap.of` powinno zadziałać

Reply:
true